go/vt/srvtopo/discover_test.go

```diff
@@ -84,28 +81,34 @@ func TestFindAllTargets(t *testing.T) {
 				},
 			},
 		},
-	}); err != nil {
-		t.Fatalf("can't add srvKeyspace: %v", err)
-	}
+	}), "can't add srvKeyspace: %v", err)
 
 	// Get it.
-	ks, err = FindAllTargets(ctx, rs, "cell1", []topodatapb.TabletType{topodatapb.TabletType_PRIMARY})
-	if err != nil {
-		t.Errorf("unexpected error: %v", err)
-	}
-	if !reflect.DeepEqual(ks, []*querypb.Target{
+	ks, err = FindAllTargets(ctx, rs, "cell1", []string{"test_keyspace"}, []topodatapb.TabletType{topodatapb.TabletType_PRIMARY})
+	assert.Nil(t, err)
+	assert.Equal(t, []*querypb.Target{
```

Comment:
We should use `assert.EqualValues()` instead here and everywhere else we're updating as that's the replacement for `reflect.DeepEqual()`.